compiler/compiler.go

```diff
@@ -113,6 +116,12 @@ func (c *Compiler) Compile(ctx context.Context, file *source.File, opts ...Optio
 					if c.autoImport && strings.HasSuffix(p, grpcFederationFilePath) {
 						return io.NopCloser(bytes.NewBuffer(federation.ProtoFile)), nil
 					}
+					if c.autoImport && strings.HasSuffix(p, googleRPCCodeFilePath) {
```

Comment:
Now that the plugins do not work without the auto import option enabled, do we need it in the first place...? In other words, are there any cases we need to disable the option?

Reply:
It was possible to disable it if someone do not want implicit dependency resolution.
It may seems good autoImport enabled by default.

Reply:
Makes sense. I've enabled the auto import by default 🙂